<launch>
    <arg name="dev_name" default="Movement"/>
    <arg name="imu_name" default="Imu"/>
    <node pkg="ros_plain_serial" name="ros_serial_TF2" type="ros_serial_TF2.py" required="true" output="screen">
        <param name="device_name" value="$(arg dev_name)"/>
        <param name="port_name" value="ttyUSB"/>
    </node>
    <node pkg="ros_plain_serial" name="ros_imu_converter" type="ros_imu_converter.py" required="true" output="screen">
        <param name="device_name" value="$(arg imu_name)"/>
        <param name="port_name" value="ttyUSB"/>
    </node>
</launch>
